src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [1, 1] if animalAge < first + second', () => {
+    expect(getHumanAge(20, 20)).toEqual([1, 1]);
```

Comment:
The test description mentions 'animalAge < first + second', but the test case uses 'getHumanAge(20, 20)'. Ensure that the function logic and test case align with the intended condition.